dotnet/src/SemanticKernel.Abstractions/Functions/KernelFunctionMetadata.cs

```diff
@@ -74,6 +77,18 @@ public string Description
         init => this._description = value ?? string.Empty;
     }
 
+    /// <summary>Gets the attributes on the function.</summary>
+    /// <remarks>If the function has no attributes, the returned list will be empty.</remarks>
+    public IReadOnlyList<Attribute> Attributes
```

Comment:
Would it not make more sense to just expose the raw `MethodInfo` instance, rather than just the attributes?